core/toolbox.js

```diff
@@ -533,7 +533,9 @@ Blockly.Toolbox.TreeNode = function(toolbox, html, opt_config, opt_domHelper) {
   goog.ui.tree.TreeNode.call(this, html, opt_config, opt_domHelper);
   if (toolbox) {
     var resize = function() {
-      Blockly.asyncSvgResize(toolbox.workspace_);
+      // Even though the div hasn't changed size, the visible workspace
+      // surface of the workspace has, so we may need to reposition everything.
+      Blockly.svgResize(Blockly.getMainWorkspace());
```

Comment:
Isn't safer to use the workspace attached to the toolbox rather than the most recently used workspace, as the original code did?